core/src/lib.rs

```diff
@@ -81,15 +81,13 @@ mod tests {
     use std::mem::size_of;
 
     use super::*;
-    use crate::raw::*;
 
     /// This is not a real test case.
     ///
     /// We assert our public structs here to make sure we don't introduce
     /// unexpected struct/enum size change.
     #[test]
     fn assert_size() {
-        assert_eq!(104, size_of::<AccessorInfo>());
```

Comment:
Is there any chance to keep this?

Reply:
No, we don't need to track AccessorInfo's size.